modules/authorization/policy-assignments/management-group/metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://aka.ms/bicep-registry-module-metadata-file-schema-v2#",
+  "name": "Policy Assignment on Management Group level",
+  "summary": "With this module you can perform policy assignments on a management group level.",
```

Comment:
Should the README.md be updated to "This module deploys policy assignments on a management group level." and then this file updated accordingly?